tests/test_components/test_IO.py

```diff
@@ -187,7 +192,10 @@ def test_validation_speed(tmp_path):
 @pytest.mark.parametrize("sim_file", SIM_FILES)
 def test_simulation_updater(sim_file):
     """Test that all simulations in ``SIM_DIR`` can be updated to current version and loaded."""
-    sim_updated = td.Simulation.from_file(sim_file)
+    if sim_file[-2:] == "h5":
```

Comment:
we could also consider just changing `from_file` to handle `.h5` by adding it here:
https://github.com/flexcompute/tidy3d/blob/develop/tidy3d/components/base.py#L223-L228

probably better while we're at it.

Reply:
Updated, have a quick look?